### ISSUE TITLE
Finish Chaosnet install script

### ISSUE THREAD

**Issue by larsbrinkhoff** [Chaosnet/Chaosnet-for-4.1BSD#4]

**Comment by ams** [comment]
I can do that.  With a bit of help from my friends... 

How do I untar a tape?  Simplest would be to have two tape drives, tu0 and tu1?

**Comment by larsbrinkhoff** [comment]
I think you got the tar tape extracted now?

**Comment by larsbrinkhoff** [comment]
You can compare your edits to mine:  
http://lars.nocrew.org/tmp/Chaotic-4.1BSD.tar.bz2

**Comment by ams** [comment]
> I think you got the tar tape extracted now?

Yes, using ugly means -- still wasn't able to figure out how to access TU1 instead.  Thanks for the tarball, that will be most useful, since I suspect you've done a bunch of changes in there then I won't need to replicate that.

**Comment by larsbrinkhoff** [comment]
It's more of a disk image.

My changes *should* be as per the instructions.
